--- Makefile.msc.old	2013-04-14 02:54:57.000000000 +0200
+++ Makefile.msc	2016-08-08 14:18:56.682204300 +0200
@@ -26,10 +26,10 @@
 LD = link
 AR = lib
 RC = rc
-CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC)
-WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE
+CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC) /Zc:threadSafeInit-
+WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE /D "_USING_V110_SDK71_"
 ASFLAGS = -coff -Zi $(LOC)
-LDFLAGS = -nologo -debug -incremental:no -opt:ref
+LDFLAGS = -nologo -debug -incremental:no -opt:ref /SUBSYSTEM:CONSOLE",5.01"
 ARFLAGS = -nologo
 RCFLAGS = /dWIN32 /r
 
